config.yaml

```diff
@@ -8,6 +8,9 @@
 # - project/config.local.yaml          (global overrides)
 # - project/config.yaml                 (global defaults)
 
+# Add a version for Chassis for when we upgrade Ubuntu distributions. We're currently on version 2.1: https://github.com/Chassis/Chassis/releases/tag/2.1
+version: 2.1
```

Comment:
Should this be split out? 🤔

Reply:
I thought about that but seeing we've been slack with version I figured we can slip it in. 🍆